eng/deploy/azure/README.md

```diff
@@ -63,6 +69,43 @@ az container create --resource-group $resourceGroup -n ml-api `
     ALLOW__EXT_PROPERTY=true MONGO_URI=$mongoUri
 ```
 
+### Enable Logging
+
+To save the logs to a file, for a summarized result, set the flag `SAVE_LOG_TO_FILE` to true, which will create a `meadowlark.log` file with the logs.
+
+For a production deployment, it's recommended to send the logs to _Log Analytics_, with the following steps:
+
+```Shell
```

Comment:
Recommend changing to "pwsh" since this code is PowerShell specific and would fail in Bash (the line continuation character).